CMakeLists.txt

```diff
@@ -47,18 +47,13 @@ endif()
 
 # Assume MSVC projects don't have a package manager and need vendored dependencies (by default).
 # Most other platforms have some kind of package manager.
-if(ANDROID OR MSVC AND NOT VCPKG_TOOLCHAIN AND NOT CONAN_CMD)
+# FIXME: consider a package manager such as conan/vcpkg instead of vendoring
```

Comment:
I'm going to remove this comment, because it doesn't look like we'll use a package manager ourselves anytime soon.